mudz.Core/Domain/GameEngine/Handler/CommandHandler.cs

```diff
@@ -5,73 +5,79 @@
 
 namespace mudz.Core.Domain.GameEngine.Handler
 {
-    public class CommandHandler : BaseHandler
-    {
-        public override GameResponse HandleRequest(GameResponse gameResponse)
-        {
-            if(gameResponse.Request.GameAction == GameActions.Login) return gameResponse;
+	public class CommandHandler : BaseHandler
+	{
+		public override GameResponse HandleRequest(GameResponse gameResponse)
```

Comment:
This is the rendering fix here. Apparently I hit format document and messed with the spacing, sorry!